charts/jaeger/templates/spark-cronjob.yaml

```
@@ -42,6 +42,7 @@ spec:
               - name: STORAGE
                 value: {{ .Values.storage.type }}
               {{- include "storage.env" . | nindent 14 }}
+              {{- toYaml .Values.spark.env | nindent 14 }}
```

```suggestion
              {{- toYaml .Values.spark.extraEnv | nindent 14 }}
```
opening this suggestion as per https://github.com/jaegertracing/helm-charts/issues/120#issuecomment-643047774 which is still open to discussion of course :)